test/package-set.dhall

```diff
@@ -2,7 +2,7 @@
   {
     name = "matchers",
     repo = "https://github.com/kritzcreek/motoko-matchers.git",
-    version = "v1.1.0",
+    version = "90c04bf99fb156bb0dc9fa8a434347a635ff3a2f",
```

Comment:
Is this file still needed? @christoph-dfinity
If so, bump to the upcoming release...